ui/app/src/components/DashboardList.tsx

```diff
@@ -49,31 +49,33 @@ function DashboardList(props: DashboardListProperties) {
       <List>
         {dashboardList.map((dashboard, i) => {
           return (
-            <>
+            <Fragment key={dashboard.metadata.name}>
```

Comment:
Is this a safe value to use for a key? I _think_ they're unique?

Reply:
The combination `metadata.project` + `metadata.name` gives you a unique key

Reply:
`metadata.name` is unique in a given project